app/recipes/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from recipes.models import Recipe, Tag
+
+# Register your models here.
+admin.site.register(Recipe)
```

Comment:
Try to register custom admin models via decorators also, just to know how to do this